### ISSUE TITLE
better functionality in new-external new file form

### ISSUE THREAD

**Issue by GeoffFroh** [densho/ddr-local#245]
currently does not work as desired. incomplete data at entry

Froh and phil will look at this later

[ne-external new file form.txt](https://github.com/densho/ddr-local/files/2912679/ne-external.new.file.form.txt)

**Comment by GeoffFroh** [comment]
See: `forms.files.NewExternalFileForm` (https://github.com/densho/ddr-local/blob/master/ddrlocal/webui/forms/files.py#L35) for datafields:

```
class NewExternalFileForm(forms.Form):
    filename = forms.CharField(required=True, max_length=255)
    mimetype = forms.ChoiceField(choices=MIMETYPE_CHOICES, required=True)
    size = forms.CharField(required=True, max_length=255)
    sha1 = forms.CharField(required=True, max_length=255)
    md5 = forms.CharField(required=True, max_length=255)
    sha256 = forms.CharField(required=True, max_length=255)
```
Template is at: https://github.com/densho/ddr-local/blob/master/ddrlocal/webui/templates/webui/files/new-external.html

**Comment by GeoffFroh** [comment]
Initial fix in 0218658ee1ad8f40f0deacbb3c5dfaff6a88c495
